PrivateCollections/Germany/Neukirchen005.xml

```diff
@@ -167,13 +213,26 @@ schematypens="http://relaxng.org/ns/structure/1.0"
 
                   <bindingDesc>
                      <binding contemporary="true" xml:id="binding">
-                        <decoNote xml:id="b1">Two wooden boards covered with blind-tooled leather. Purple textile inlays.</decoNote>
-                        <decoNote xml:id="b2" type="bindingMaterial">
+                        <decoNote xml:id="b1">Two wooden boards of somewhat irregular size,
+                          cut not exactly along the grain but somewhat diagonally,
+                          discolored, with traces of adhesives on the inner sides.</decoNote>
+                        <decoNote xml:id="b2" type="Cover">The boards are partly covered with
+                          untooled leather (quarter cover), with turn-ins
+                        held together by threads.</decoNote>
+                        <decoNote xml:id="b3">Four pairs of sewing stations. The sewing is loose,
```

Comment:
"Four pair of sewing stations" means
`                        <decoNote xml:id="b***" type="SewingStations">8</decoNote>`

(4 x 2 = 8)